submissions/mustbefail/dom-api-task/src/js/app.js

```diff
@@ -0,0 +1,54 @@
+import makeModal from './modal.js';
+
+const makeMsg = (ticketsAmount, totalPrice) => {
+  if (!ticketsAmount) return 'No seat selected';
+  return `You have selected ${ticketsAmount} seats for a price of $${totalPrice}`;
+};
+
+const render = ({ ticketsAmount, totalPrice }) => {
+  const cartTip = document.querySelector('.add-to-cart-block p');
+  cartTip.textContent = makeMsg(ticketsAmount, totalPrice);
+};
+
+export default () => {
```

Comment:
Export default, not a good practice, use single export. [Read more](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#import_a_single_export_from_a_module)